CONTRIBUTING.md

```diff
@@ -26,10 +32,26 @@ The last option is needed because our guides feature code examples. **markdown-s
 
 For now it'll simply report all errors and you can then deal with actually spelling errors directly.
 
-We use [markdownlint](https://github.com/mivok/markdownlint) to check the files have a consistent style as part of a CI process. We check contributions using `mdl --style md_style.rb $PWD`. We recommend you install this and run it before pushing your commits.
+We use [markdownlint](https://github.com/mivok/markdownlint) to check the files have a consistent style as part of a CI process. We check contributions using `mdl --style md_style.rb $PWD`. We recommend you install this and run it before requesting feedback.
+
+## When you're finished
+
+Make sure everything is pushed, then add the **ready for review** label to the PR and ensure you are assigned. If you have access to it announce your PR in our [Slack](https://defra-digital.slack.com/) [#dst-guides](https://defra-digital.slack.com/messages/dst-guides/) channel.
+
+If you don't add a comment mentioning either **@cruikshanks** or **@bensagar-ea**.
 
 ## Getting feedback
 
-Feedback will be provided by the team within the PR. Decisions will be documented and reasons given.
+Feedback will be provided within the PR. Decisions will be documented and reasons given.
+
+We want to encourage all to contribute and suggest ways we can improve. Anyone providing feedback should be positive and professional with any comments made about your contribution, but let us know if you feel we failed.
+
+We don't expect everyone to be available or able to contribute but we want to allow enough time for those who wish to get involved. Don't merge your PR for at least 3 days, even if you've had lots of positive feedback.
+
+## Finishing up
+
+Leave the PR open for at least 3 working days and ensure at least one other person has reviewed it.
+
+Once those who have participated in the PR have said they are happy, first [squash]() your commits down to a single commit (checkout the **Completing the commit** section in [Pull requests](/process/pull_request.md) if you are unsure how to do this). Then merge it in GitHub (not locally). Don't forget to also delete the branch in GitHub.
```

Comment:
It would be nice if we could adopt a process here that is friendly to those not super-familiar with Git; this shouldn't only be for software developers but for the whole team.  Command-line squashing and rebasing doesn't seem a brilliant fit.